src/clj/markdown/core.clj

```diff
@@ -62,7 +62,9 @@
   (when (not= 0 (mod (count params) 2))
     (throw (IllegalArgumentException.
              "Must supply an even number of parameters")))
-  (when params (apply assoc {} params)))
+  (some->> params
```

Comment:
This bit isn't needed because this would be called from Clojure on the JVM runtime.

Reply:
true... I tried to skip this part, but then I got test failures. Forgive my clojurescript building ignorance, but is there a way to say "this test is only for cljs?"